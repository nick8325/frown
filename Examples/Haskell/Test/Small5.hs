f4 x = s where {
s = 12
}
