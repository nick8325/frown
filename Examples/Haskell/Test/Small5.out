module Main where
f4 x = s
  where s = 12
